OpenRA.Game/Settings.cs

```diff
@@ -288,6 +288,19 @@ Hotkey GetKey(Func<KeySettings, Hotkey>[] keys, int index)
 		}
 	}
 
+	public class IrcSettings
+	{
+		public string[] Hostname = { "irc.openra.net" };
+		public int Port = 6667;
+		public string Nickname = "Newbie";
+		public string Username = "OpenRA";
```

Comment:
Do we have a reason to actually expose both of these?
Changing nickname in IRC (if possible) ought to change username, which shouldn't matter at all to players / people chatting.